### DISCUSSION TITLE
A swipe feature to move between Home, Folders, Playlist, Tracks.

### DISCUSSION THREAD

**Discussion by adarshtr0011** [MissingCore/Music#76] [Ideas]
By just swiping to the left or right if it was able to move between these folders it will be very good and functional.

**Comment by Ender-Wang** [comment]
And it's intuitive too, I thought I could do that when I opened it the first time.

**Comment by cyanChill** [comment]
Home screen swipe navigation is coming in the next major update, if you're using the APK version, you can try it out early with the [`v2.0.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.0.0-rc.1) release.
